fastify.js

```diff
@@ -293,7 +297,9 @@ function build (options) {
 
     req.log.info({ req }, 'incoming request')
 
-    var request = new context.Request(params, req, urlUtil.parse(req.url, true).query, req.headers, req.log)
+    var queryPrefix = req.url.indexOf('?')
+    var query = queryPrefix !== -1 ? querystringParser(req.url.slice(queryPrefix + 1)) : {}
```

Comment:
Yup. Although the second part of the ternary would be a string:

```suggestion
    var query = querystringParser(queryPrefix > -1 ? req.url.slice(queryPrefix + 1) : '')
```